mathesar_ui/src/pages/schema/SchemaPage.svelte

```diff
@@ -81,6 +83,13 @@
   }
 
   $: isDefault = schema.name === 'public';
+  onMount(async () => {
+    logEvent('opened_schema', {
+      database_name: database.name,
+      schema_name: schema.name,
```

Comment:
This does not have to be placed within onMount.

`onMount` only gets called after the component is rendered to the DOM. The `logEvent` function has nothing to do with the DOM, so we can directly specify it on the component.